projects/aca-content/assets/i18n/en.json

```diff
@@ -463,7 +463,13 @@
             "COLLAPSE_NAVIGATION": "Collapse navigation menu",
             "OPTIONS_SETTINGS": "Options and settings",
             "MY_PROFILE": "My profile",
-            "EXPAND_NAVIGATION": "Expand navigation menu"
+            "EXPAND_NAVIGATION": "Expand navigation menu",
+            "NAVIGATE_TO_PERSONAL_FILES": "Navigate to personal files",
```

Comment:
I can't see usages for that translations keys: NAVIGATE_TO_PERSONAL_FILES. Is it used or can be removed?